mica/archive/aca_dark/dark_cal.py

```diff
@@ -9,12 +9,12 @@
 import pyyaks.context
 from Chandra.Time import DateTime
 
+from chandra_aca.dark_model import dark_temp_scale
```

Comment:
Searching in the `sot` org or recursively grepping in  our dependencies/repos area is a decent start for finding dependencies.